src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,51 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<I, S> {
+    private I first;
+    private S second;
 
+    private Pair(I first, S second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public I getFirst() {
+        return first;
+    }
+
+    public S getSecond() {
+        return second;
+    }
+
+    public void setFirst(I first) {
+        this.first = first;
+    }
+
+    public void setSecond(S second) {
+        this.second = second;
+    }
+
+    public static <I, S> Pair<I,S> of(I first, S second) {
+        return new Pair<I, S>(first, second);
```

Comment:
```suggestion
        return new Pair<>(first, second);
```
writing just `<>` here is possible because of the [diamond operator](https://www.baeldung.com/java-diamond-operator)